From e40605dad11724885a9a2c89585971f00f0a5e14 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:33:02 -0400
Subject: [alsa] lola: fix format type mismatch in sound/pci/lola/lola_proc.c

Message-id: <140972823573-306-patchset-jkysela@redhat.com>
Patchwork-id: 91002
O-Subject: [RHEL 7.1 PATCH 305/442] ALSA: lola: fix format type mismatch in sound/pci/lola/lola_proc.c
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Signed-off-by: Toralf Foerster <toralf.foerster@gmx.de>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 2d3a27782259de867fd2c0b7e3428acaf2b00123)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/lola/lola_proc.c b/sound/pci/lola/lola_proc.c
index 04df83d..c241dc0 100644
--- a/sound/pci/lola/lola_proc.c
+++ b/sound/pci/lola/lola_proc.c
@@ -151,7 +151,7 @@ static void lola_proc_codec_rw_write(struct snd_info_entry *entry,
  char line[64];
  unsigned int id, verb, data, extdata;
  while (!snd_info_get_line(buffer, line, sizeof(line))) {
-  if (sscanf(line, "%i %i %i %i", &id, &verb, &data, &extdata) != 4)
+  if (sscanf(line, "%u %u %u %u", &id, &verb, &data, &extdata) != 4)
    continue;
   lola_codec_read(chip, id, verb, data, extdata,
     &chip->debug_res,
-- 
1.7.1